serverless-workflow-operator/pkg/controller/workflow/workflow_controller.go

```diff
@@ -320,3 +342,56 @@ func getImage(cr *v1.Workflow) string {
 func intPtr(x int32) *int32 {
 	return &x
 }
+
+func newKnativeService(cr *v1.Workflow) *knsv1.Service {
+	return &knsv1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cr.Name,
+			Namespace: cr.Namespace,
+			Labels:    getLabels(cr),
+		},
+		Spec: knsv1.ServiceSpec{
+			ConfigurationSpec: knsv1.ConfigurationSpec{
+				Template: knsv1.RevisionTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      cr.Name + "-v1",
```

Comment:
Why -v1?

Reply:
It is the revision of the revision template. At least, v1alpha1 requires the revision in the revision template name.
But it is not mandatory to assign a name to the revision template. I'll remove the name, and a revision will be generated by knative serving.

Reply:
Exactly. KNative will create a revision accordingly